pandas/tests/io/test_pickle.py

```diff
@@ -552,21 +552,30 @@ def test_pickle_binary_object_compression(compression):
     tm.assert_frame_equal(df, read_df)
 
 
-def test_pickle_dataframe_with_multilevel_index(
-    multiindex_year_month_day_dataframe_random_data,
-    multiindex_dataframe_random_data,
-):
-    ymd = multiindex_year_month_day_dataframe_random_data
-    frame = multiindex_dataframe_random_data
-
-    def _test_roundtrip(frame):
-        unpickled = tm.round_trip_pickle(frame)
-        tm.assert_frame_equal(frame, unpickled)
-
-    _test_roundtrip(frame)
-    _test_roundtrip(frame.T)
-    _test_roundtrip(ymd)
-    _test_roundtrip(ymd.T)
+@pytest.mark.parametrize(
+    "fixture_name",
+    [
+        "multiindex_year_month_day_dataframe_random_data",
+        "multiindex_dataframe_random_data",
+    ],
+)
+def test_pickle_dataframe_with_multilevel_index(fixture_name, request):
+    frame = request.getfixturevalue(fixture_name)
```

Comment:
might just be me, but i find this indirection harder to follow than the status quo